### PR TITLE
Adding the 'Retrieve and open' option

### PR DESCRIPTION
**Pull request by LisandroFernandezSF**
This commit adds a new button and the corresponding explanation to the ORG Browser layout. Point 9

### What does this PR do?
This PR adds the documentation needed to use the new feature to download and open metadata items from ORG Browser.

### What issues does this PR fix or reference?
W-8101795

### CODE REVIEW

File: docs/_articles/en/user-guide/org-browser.md [hunk 1]
```diff
@@ -17,6 +17,7 @@ The Org Browser displays the available metadata types and their corresponding co
 6. Refresh components for metadata type
 7. Retrieve source for metadata component
 8. Retrieve source for all components of metadata type
+9. Retrieve source for metadata component and open it on VSCode
```
Comment (ekapner) [inline docs/_articles/en/user-guide/org-browser.md:20]: 9. Retrieve source for metadata component and open it in VS Code.

**Review by ekapner — COMMENTED** [review]
Small edit to text

File: docs/_articles/en/user-guide/org-browser.md (hunk 1 above)
Comment (LisandroFernandezSF) [reply docs/_articles/en/user-guide/org-browser.md]: Done! 
I'm not using the final point to keep the coherence with the previous bullets.
@ekapner Let me know if it's fine

Comment (ekapner) [reply docs/_articles/en/user-guide/org-browser.md]: Yes! @LisandroFernandezSF - no ending period needed. Whoops.

**Review by lcampos — APPROVED** [review]

**Pull request merged**